Bojzen-I-Mitten/Stort-spelprojekt
# Pickup text

* Added fadeable pickup text for objects (ball, powerups)
* Added text outline support
* Added disable rendering text support

*** HOW TO TEST ***
* Start up a match and try picking up an object to see a pickup text appear and fade away after
a short delay
